templates/terraform/sweeper_file.go.erb

```diff
@@ -0,0 +1,115 @@
+<%= lines(autogen_notice :go) -%>
+
+package google
+
+import (
+  "fmt"
+  "testing"
+  "strings"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+  "github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+<%
+sweeper_name = product_ns + object.name
+wrap_path = object&.nested_query&.keys&.first || object.collection_url_key
+-%>
+
+func init() {
+	resource.AddTestSweepers("<%= sweeper_name -%>", &resource.Sweeper{
+		Name: "<%= sweeper_name -%>",
+		F:    testSweep<%= sweeper_name -%>,
+	})
+}
+
+func testSweep<%= sweeper_name -%>(region string) error {
+	resource_name := "<%= sweeper_name -%>"
+	log.Printf("[SWEEP_LOG] Sweeping %s",resource_name)
+
+	config, err := sharedConfigForRegion(region)
+	if err != nil {
+		log.Fatalf("[SWEEP_LOG] error getting shared config for region: %s", err)
```

Comment:
What's the difference between calling log.Fatal and returning an error?

Reply:
Not much, returning the error is probably better. Just left that in from some of the handwritten sweepers